### PR TITLE
chore: try add a dependabot

### LINKED ISSUE #40: ci: Add dependabot to make sure dependences are up to date
Let's add dependabot config to make sure dependences are up to date. To avoid DDoS our maintianers, we can set the internal to one month.

### PR DESCRIPTION
**Pull request by PsiACE**
Close #40

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    open-pull-requests-limit: 1
+    schedule:
+      interval: "monthly"
+
+  - package-ecosystem: "cargo"
```
Comment (Xuanwo) [inline .github/dependabot.yml:26]: I feel like we should add this for every rust/python/java package?

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (Xuanwo) [inline .github/dependabot.yml:1]: The license seems wrong

Comment (PsiACE) [reply .github/dependabot.yml]: fixed in [34c1004](https://github.com/risingwavelabs/arrow-udf/pull/42/commits/34c1004bea9224bb21317cd2a44fbf534e8a7326)

File: .github/dependabot.yml (hunk 1 above)
Comment (PsiACE) [reply .github/dependabot.yml:23]: add maven and pip in [ff0a1c2](https://github.com/risingwavelabs/arrow-udf/pull/42/commits/ff0a1c23156d521a89132dfc4a323aa677fe6488)

**Pull request merged**